acceptance/tests/resource/user/should_purge.rb

```diff
@@ -0,0 +1,40 @@
+test_name "should purge a user"
+confine :except, :platform => /^eos-/ # See ARISTA-37
+confine :except, :platform => /^cisco_/ # See PUP-5828
+tag 'audit:medium',
+    'audit:refactor',  # Use block style `test_run`
```

Comment:
Don't need this -- the `block_style 'test_run'` is what I was referring to in my preceding comment.

Reply:
This is also standard throughout the `user` directory, which seems to be outdated.

Reply:
Yup. I ran into that same issue in an earlier PR so thought I'd let you know about it.